src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -506,6 +510,36 @@ public void delete() throws IOException {
     }
 
     /**
+     * Sort orders for listing forks
+     */
+    public static enum Sort { NEWEST, OLDEST, STARGAZERS }
```

Comment:
Again, I looked how similar stuff was implemented, so yes,  I know there are 3 other Sort enums. Mine is also the only one with javadoc.